google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -300,6 +302,11 @@ public void addSupport(@NotNull Module module,
       AppEngineSupportProvider.this
           .addSupport(module, rootModel, myFrameworkSupportModel,
               PersistenceApiComboboxUtil.getSelectedApi(myPersistenceApiComboBox));
+
+      // Called on New Project or New Module dialogs.
```

Comment:
go to a module root in the left nav, right click on it, and select 'Add
Framework Support..'

On Wed, Aug 31, 2016 at 3:30 PM, João Martins <notifications@github.com>
wrote:

> In google-cloud-tools-plugin/src/com/google/cloud/tools/
> intellij/appengine/facet/AppEngineSupportProvider.java
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/850#discussion_r77056752>
> :
>
> > @@ -300,6 +302,11 @@ public void addSupport(@NotNull Module module,
> >        AppEngineSupportProvider.this
> >            .addSupport(module, rootModel, myFrameworkSupportModel,
> >                PersistenceApiComboboxUtil.getSelectedApi(myPersistenceApiComboBox));
> > +
> > +      // Called on New Project or New Module dialogs.
>
> By adding framework support, do you mean going to Project Settings and
> adding the GAE facet there? Or clicking the Framework discovered Configure
> link?
>
> Doing this doesn't trigger addSupport() for me. Let me know if there's
> another way I'm overlooking.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/850/files/b9ac4f354079a5cd7527300acea5b9b682844bc3#r77056752>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHf5Ha2ur-Y1Flw8V2fVYZhUM9iDeadEks5qldZpgaJpZM4Jvw29>
> .
>